utils/labeller/index.js

```diff
@@ -154,9 +155,24 @@ function github_process_prs(res, pr_cb){
                     }
                   });
                 });
+                
+                if (!fs.existsSync('../../' + file.filename)) {
```

Comment:
In most contexts this is run, this won't exist.  The README suggests running this script from within `docker`, which lacks the context of the rest of the project.  Also, this will only work if you're doing a `git pull` before each run.  There must be a way within the github api to tell if a file is new or not.